.github/workflows/diff_report.yml

```diff
@@ -204,10 +213,10 @@ jobs:
      
       - name: Set message
         id: out
-        run: echo ::set-output name=message::$(cat message)   
-      
```

Comment:
revert from this file all formatting/spaces changes.

Reply:
Intellij automatically removes these spaces at end of the sentence and in a new line.

Reply:
please update you IDEA to not do this.
if you want to keep formatting, it MUST be done in separate commit with message "minor: formatting by IDEA" of this PR

Reply:
Got it. Solved.